doc/manual/manual.tex

```diff
@@ -4701,41 +4701,41 @@ \subsubsection{Convection in a 3d box}
-\lstinputlisting[language=prmfile]{cookbooks/convection-box-3d/doc/start.part.prm.out}
+\lstinputlisting[language=prmfile]{cookbooks/convection\_box\_3d/doc/start.part.prm.out}
 
 
 The next step is to describe the initial conditions. As before, we will use an
 unstably layered medium but the perturbation now needs to be both in $x$- and
 $y$-direction
-\lstinputlisting[language=prmfile]{cookbooks/convection-box-3d/doc/initial.part.prm.out}
+\lstinputlisting[language=prmfile]{cookbooks/convection\_box\_3d/doc/initial.part.prm.out}
 
 The third issue we need to address is that we can likely not afford a mesh as
 fine as in 2d. We choose a mesh that is refined 3 times globally at the
 beginning, then 3 times adaptively, and is then adapted every 15 time steps. We
 also allow one additional mesh refinement in the first time step following
 $t=0.003$ once the initial instability has given way to a more stable pattern:
-\lstinputlisting[language=prmfile]{cookbooks/convection-box-3d/doc/amr.part.prm.out}
+\lstinputlisting[language=prmfile]{cookbooks/convection\_box\_3d/doc/amr.part.prm.out}
 
 Finally, as we have seen in the previous section, a computation with $Ra=10^4$
 does not lead to a simulation that is exactly exciting. Let us choose $Ra=10^6$
 instead (the mesh chosen above with up to 7 refinement levels after some time
 is fine enough to resolve this). We can achieve this in the same way as in the
 previous section by choosing $\alpha=10^{-10}$ and setting
-\lstinputlisting[language=prmfile]{cookbooks/convection-box-3d/doc/gravity.part.prm.out}
+\lstinputlisting[language=prmfile]{cookbooks/convection\_box\_3d/doc/gravity.part.prm.out}
 This has some interesting implications. First, a higher Rayleigh number makes
 time scales correspondingly smaller; where we generated graphical output only
 once every 0.01 time units before, we now need to choose the corresponding
 increment smaller by a factor of 100:
-\lstinputlisting[language=prmfile]{cookbooks/convection-box-3d/doc/postprocess.part.prm.out}
+\lstinputlisting[language=prmfile]{cookbooks/convection\_box\_3d/doc/postprocess.part.prm.out}
 Secondly, a simulation like this -- in 3d, with a significant number of cells,
 and for a significant number of time steps -- will likely take a good amount of
 time. The computations for which we show results below was run using 64
 processors by running it using the command
-{\tt{mpirun -n 64 ./aspect convection-box-3d.prm}}. If the machine should crash
+{\tt{mpirun -n 64 ./aspect convection\_box\_3d.prm}}. If the machine should crash
 during such a run, a significant amount of compute time would be lost if we had
 to run everything from the start. However, we can avoid this by periodically
 checkpointing the state of the computation:
-\lstinputlisting[language=prmfile]{cookbooks/convection-box-3d/doc/checkpoint.part.prm.out}
+\lstinputlisting[language=prmfile]{cookbooks/convection\_box\_3d/doc/checkpoint.part.prm.out}
```

Comment:
I think for lstinputlisting you do must not escape them, so this was correct earlier.